<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="ScansViewer" name="scansViewer">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>411</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Compatible devices:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="computingDevicesComboBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>400</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="ScansDataTree" name="scansDataTree">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addPatientButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="removePatientButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ScansDataTree</class>
   <extends>QTreeView</extends>
   <header>scans_data_tree.h</header>
  </customwidget>
  <customwidget>
   <class>ScansViewer</class>
   <extends>QVTKWidget</extends>
   <header>scans_viewer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
